iqmotion/iq_devices/iq_module.py

```diff
@@ -20,23 +20,33 @@ class IqModule:
     _DEFAULT_VELOCITY_CLIENT_ENTRY = ""
     _DEFAULT_VOLTS_CLIENT_ENTRY = ""
 
-    def __init__(self, com: Communicator, module_idn=0, clients_path=None):
+    def __init__(self, com: Communicator, 
+                module_idn=0,  
+                clients_path: str=None, 
+                extra_clients: list=None, 
+                module_file_path: str=None):
         
         self._client_dict = {}
         self._com = com
         self._module_idn = module_idn
 
         # Load the Module JSON (Base, Vertiq, Fortiq, etc...)
-        self._module_file_dict = IqModuleJsonParser.parse_default_modules(
-            self._MODULE_FILE_NAME
-        )
-        
-        # Deprecated: Add a custom module 
-        # self._module_file_dict = IqModuleJsonParser.parse_module(module_file_path)
+        if module_file_path is None:
```

Comment:
If custom module is out now, do you still need this bit?

Reply:
This could be removed if we plan on completely deprecating custom_iq_module. This would have to be done on the bitbucket repo first though